### DISCUSSION TITLE
Seplos BMS V3 in daisy chain (3 units)

### DISCUSSION THREAD

**Discussion by cristof48** [syssi/esphome-seplos-bms#90] [General]
Hello,
I participate in the “PVbrain2” project with French Esphome team
I have 3 SEPLOS BMS V3 batteries connected in a daisy chain for monitoring via RS485 to BMS studio. I would like to be able to send the 3 batteries to a single uart in the project.
When they are in daisy chain, the assignment of modbus addresses is done automatically modbus address 0 for battery master 1, modbus 2 for battery 2 and so.
Could you consider writing the code for the Seplos V3 BMS in order to be able to receive simultaneously on a single uart from an esp32?
THANKS.

**Comment by syssi** [comment]
I've prepared an basic example here: https://github.com/syssi/esphome-seplos-bms/pull/91/files

Please give it a try!
